catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -211,6 +211,11 @@
         <bundle>mvn:ddf.catalog.transformer/video-input-transformer/${project.version}</bundle>
     </feature>
 
+    <feature name="catalog-transformer-microsoft" install="manual" version="${project.version}"
+             description="Microsoft Word Input Transformer.">
```

Comment:
The name of this input transformer makes it sound like it is for all microsoft file types, but the description restricts it to Word docs. Should the name be catalog-transformer-msword?